app/main.py

```diff
@@ -1,3 +1,44 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f'Distance: {self.km} kilometers.'
+
+    def __repr__(self):
+        return f'Distance(km={self.km})'
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            km = self.km + other.km
+        else:
+            km = self.km + other
+        return Distance(km)
+
+    def __iadd__(self, other):
+        self.km = self.__add__(other).km
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other):
+        return self.km < other
```

Comment:
What if `other` is of type `Distance` here and in all the following methods?

Reply:
Thank you! Somehow it slipped out of my mind to create a separate static method for the ckecks.